### PR TITLE
[99] BigQuery integration

### PR DESCRIPTION
**Pull request by AbigailMcP**
### Context

We want to be able to track the usage of our service. Using BigQuery is an established pattern across DfE.

It needs to be explicitly in development as you need a key from the Google Cloud console in order to use it.

Follow the instructions in the README for more detail.

### Changes proposed in this pull request

Add event tracking with BiqQuery / dfe-analytics
Remove the active_storage tables that were generated automatically and aren't needed for this project

### Link to Trello card

https://trello.com/c/d4lqiMQJ/99-add-bigquery-integration

### Checklist

- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by AbigailMcP** [comment]
[Add BigQuery integration](https://trello.com/c/d4lqiMQJ/99-add-bigquery-integration)

File: terraform/aks/variables.tf [hunk 1]
```diff
@@ -151,7 +151,13 @@ locals {
 
   app_env_values = merge(
     local.app_config,
-    { DB_SSLMODE = local.postgres_ssl_mode }
+    {
+      BIGQUERY_PROJECT_ID = "teaching-qualifications",
```
Comment (AbigailMcP) [inline terraform/aks/variables.tf:155]: I think these could also go in `terraform/aks/config/app_config.yml` 🤷‍♀️

**Review by steventux — APPROVED** [review]
:rocket:

**Review by gpeng — CHANGES_REQUESTED** [review]

File: config/analytics.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+shared:
+  :active_storage_variant_records:
+    - id
+    - blob_id
+    - variation_digest
+  :active_storage_blobs:
```
[the remark this answers is not in the document]
Comment (AbigailMcP) [reply config/analytics.yml]: gone! 4b65a8ee3068fcf01e4fb8cb0db4c46cd6f9b16c

**Review by gpeng — APPROVED** [review]

**Pull request merged**